src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -124,8 +124,17 @@ public class StripePaymentService : IPaymentService
                 Metadata = stripeCustomerMetadata,
                 InvoiceSettings = new Stripe.CustomerInvoiceSettingsOptions
                 {
-                    DefaultPaymentMethod = stipeCustomerPaymentMethodId
+                    DefaultPaymentMethod = stipeCustomerPaymentMethodId,
+                    CustomFields = new List<Stripe.CustomerInvoiceSettingsCustomFieldOptions>
+                    {
+                        new Stripe.CustomerInvoiceSettingsCustomFieldOptions()
```

Comment:
This bit of code is repeated a few times, I suggest creating a private method to retrieve the ISubscriber name and type:

```
private Stripe.CustomerInvoiceSettingsCustomFieldOptions GetSubscriberNameInvoiceCustomFieldOptions(ISubscriber subscriber)
    {
        return new Stripe.CustomerInvoiceSettingsCustomFieldOptions()
        {
            Name = subscriber.IsUser() ? "Subscriber" : "Organization", 
            Value = subscriber.SubscriberName(),
        };
    }
```